monero-outreach-docs/translations/es/archive/rpc_pay_es.md

```diff
@@ -1,113 +1,109 @@
+¿Por qué ejecutar un nodo público de Monero? Cuesta dinero, pero no hace dinero; al menos no directamente. Una preocupación recurrente es cómo incentivar más de estos nodos. RPC-Pay ofrece este incentivo y mucho más. Nos da una manera de usar los hashes en la minería de Monero, pero solo los hashes, para pagar por las llamadas de procedimiento remoto (RPC - Remote Procedure Calls) desde cualquier servidor habilitado para RPC-Pay, igual como un nodo de Monero. Los hashes recibidos como pago son usados por el servidor para obtener ingresos, y gracias RandomX de Monero, los hashes de la minería pueden ser calculados eficientemente por los CPUs más comunes. Pagar solamente con hashes en vez de Monero como tal, significa que RPC-Pay no sobrecarga la red ni deja registro alguno. Con RPC-Pay, minar hashes se convierte en la nueva unidad de pago oculta más pequeña en el universo de Monero.
 
 ### _RPC (Llamadas de procedimiento remoto)_
 
-Before studying RPC-Pay, it is helpful to understand the larger concept of RPC. RPC is a process through which one program (a client) requests data and calculations from another program (the server). The server can be on any computer on the Internet. An RPC request is like a subroutine call in a regular program except with RPC the subroutine work is performed on the server. Special intermediary software handles the network communication of both the query and its answer so that the programmer sees it as seamless. Many different types of interface software have been used to implement RPC over the years.
-Antes de estudiar sobre los Pagos RPC, es útil comprender el amplio concepto de RPC. RPC es el proceso por el cual un programa (un cliente) solícita información
+Antes de estudiar sobre qué es RPC-Pay, es útil comprender el concepto amplio de RPC. RPC es el proceso por el cual un programa (un cliente) solícita información y cálculos desde otro programa (el servidor). El servidor puede estar en cualquier computador en el Internet. Una solicitud RPC es como una llamada de subrutina en un programa normal, excepto que con el RPC, el trabajo que se ejecuta de la subrutina se realiza en el servidor. Un software intermediario específico se encarga de la red de comunicación entre la consulta y la respuesta para que el programador la va sin inconvenientes. Muchos tipos de software de interfaz han sido utilizados a través de los años para implementar llamadas RPC.
```

Comment:
"Solícita" has another meaning, for "requests" its without accent:

_RPC es el proceso por el cual un programa (un cliente) **solicita** información..._

It may make a bit more sense like this:

_...excepto que con el RPC, **el trabajo de subrutina se realiza en el servidor.**_

A correction:

_y la respuesta para que el programador la **vea** sin inconvenientes._

Reply:
Fixed and changed. :)